packages/hooks/README.md

```diff
@@ -0,0 +1,105 @@
+# Slack CLI Hooks
```

Comment:
We went back and forth a bit on this, but I prefer to have relative parity between this and what we establish as the README design proposed on the Python hooks package (absent of a good reason not to, of course).